java/src/jmri/jmrit/logix/OBlock.java

```diff
@@ -109,6 +107,26 @@ private OBlockStatus(int status, String name, String descr) {
 
     private static final Color DEFAULT_FILL_COLOR = new Color(200, 0, 200);
 
+    /**
+     * String constant to represent path State.
+     */
+    public static final String PROPERTY_PATH_STATE = "pathState";
+
+    /**
+     * String constant to represent path Count.
+     */
+    public static final String PROPERTY_PATH_COUNT = "pathCount";
+
+    /**
+     * String constant to represent portal Count.
+     */
+    public static final String PROPERTY_PORTAL_COUNT = "portalCount";
```

Comment:
No need to change, but a suggestion.

It looks better if you change State and Count in JavaDoc to lower case.

Reply:
Sure, I was thinking it may help to represent the camel case in the property value, but not the case !